packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offers/template.html

```diff
@@ -0,0 +1,64 @@
+<div class="row mb-3">
+    <oui-select-picker
+        class="col-md-4 mb-3"
+        data-ng-repeat="plan in $ctrl.backupOffers track by offerName"
+        name="plan"
+        data-model="$ctrl.selectedOffer"
+        data-label="{{:: 'dedicatedCloud_plan_' + plan.offerName | translate }}"
+        data-variant="light"
+        data-on-change="$ctrl.onSelected(modelValue)"
+        data-values="[plan]"
+    >
+        <oui-select-picker-section>
+            <ul class="plan-list-style">
+                <li
+                    data-translate="dedicatedCloud_plan_upgrade_custom_report"
+                    data-ng-if="plan.customReport"
+                ></li>
+                <li
+                    data-translate="dedicatedCloud_plan_upgrade_encrypted_backup"
+                    data-ng-if="plan.encryption"
+                ></li>
+                <li
+                    data-translate="dedicatedCloud_plan_upgrade_retention_points"
+                    data-translate-values="{
+                        retention: plan.retention,
+                    }"
+                ></li>
+                <li
+                    data-translate="dedicatedCloud_plan_upgrade_full_backups"
+                    data-translate-values="{
+                        fullBackups: plan.minimumFullBackups,
+                    }"
+                ></li>
+                <li
+                    data-translate="dedicatedCloud_plan_upgrade_replication"
+                    data-ng-if="plan.replication"
+                ></li>
+                <li
+                    data-ng-if="plan.backupDays"
+                    data-translate="dedicatedCloud_plan_upgrade_choice_backup_days"
+                ></li>
+                <li
+                    data-ng-if="plan.scheduleHour"
+                    data-translate="dedicatedCloud_plan_upgrade_choice_backup_time"
+                ></li>
+                <li
+                    data-ng-if="plan.offerName === 'premium'"
+                    data-translate="dedicatedCloud_plan_upgrade_long_term_retention"
+                ></li>
+                <li
+                    data-ng-if="plan.proxyPerHost"
+                    data-translate="dedicatedCloud_plan_upgrade_proxy_backup"
+                ></li>
+            </ul>
+        </oui-select-picker-section>
+        <oui-select-picker-footer>
+            <private-registry-price
+                data-price="plan.pricings[0].price"
```

Comment:
Only one pricing ? Maybe we should get the price according to the `capacity` instead

Reply:
Yes the pricings are based on capacity but we are supposed to show only the lowest one it seems(smallest plan). Will write the logic to pick the exact price in the wrapper class.